content/en/docs/Configuration/p8s-jaeger-grafana/tracing/tempo.md

```diff
@@ -86,6 +90,37 @@ spec:
       url: "http://my-tempo-host:3200"
 ```
 
+##### Service check URL
+
+By default, Kiali will check the service health in the endpoint `/status/services`, but sometimes, this is exposed in a different url, which can lead to a component unreachable message: 
+
+![component_unreachable](/images/documentation/configuration/component_unreachable.png)
+
+That can be changed with the `health_check_url` configuration option. 
```

Comment:
"This can be changed"

Reply:
I'll create a suggestion to make it quicker/easier for josune to incorporate this change:

```suggestion
This can be changed with the `health_check_url` configuration option. 
```